e2e/terraform/packer/ubuntu-jammy-amd64/setup.sh

```diff
@@ -63,6 +64,25 @@ sudo apt-get install -y \
      consul-enterprise \
      nomad
 
+# TODO(tgross: replace with downloading the binary from releases.hashicorp.com
+# once the official 1.4.2 release has shipped
+echo "Installing consul-cni plugin"
+sudo apt-get install -y build-essential git curl
+
+pushd /tmp
+curl -LO https://go.dev/dl/go1.22.2.linux-amd64.tar.gz
+sudo tar -C /usr/local -xzf go1.22.2.linux-amd64.tar.gz
```

Comment:
we really didn't already have Go installed on these boxes?  I'd put this under its own little heading, even if it's only currently used for "consul-k8s".

unless you're piling these things together to be also removed in bulk after 1.4.2 is shipped?

Reply:
No, we don't generally need toolchains on these machines. And yeah, I was putting them all together so they're easy to find and remove once 1.4.2 ships.